### DISCUSSION TITLE
Één

### DISCUSSION THREAD

**Discussion by Boxiet** [iptv-org/community#712] [Q&A]
**Channel Name:** ÉÉN
**Channel ID:** VRT1.be
**Country:** Belgium
**Language:** Dutch
**Website:** https://www.een.be/
**Stream URL (optional):** https://www.vrt.be/vrtnu/livestream/
**Notes (optional):** xxx

**Comment by Alstruit** [comment]
Geo-blocked and DRMed.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19691
